### ISSUE TITLE
fix UI accessibility test failure

### ISSUE THREAD

**Issue by hemarina** [Azure/awesome-azd#811]
Fix https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanReport?siteId=3d139aae-239e-4d38-a2c0-08dc2ed19924&urlId=584471
